calblueprint/meepanyar
# Tiffany/profile frontend

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

With this PR, I updated the base customer profile to fit the new Figma changes & added conditions to change the look/data displayed on the profile based on the given customer's meter type. The default is Analog Meter.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review

- Edit a customer's meter type in Airtable (or navigate to the pre-changed customers below)
- Login using testUser
- Navigate to that customer's profile screen

[//]: # 'The order in which to review files and what to expect when testing locally'

## Screenshots

Analog Meter:
![image](https://user-images.githubusercontent.com/20329981/113646643-133f8200-963e-11eb-9adf-738af587e4e6.png)

Smart Meter:
![image](https://user-images.githubusercontent.com/20329981/113646183-22720000-963d-11eb-9342-4da9786c604e.png)

No Meter:
![image](https://user-images.githubusercontent.com/20329981/113646369-8dbbd200-963d-11eb-9742-af92c8584927.png)

Inactive:
![image](https://user-images.githubusercontent.com/20329981/113646393-97453a00-963d-11eb-880b-605dafaeb6d8.png)

## Next steps

I put a few TODO's in the code, mostly about linking future pages like the create payments or edit starting meter pages which haven't been made yet. Correct me if I'm wrong!

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

CC: @julianrkung @micahtyong @wangannie 

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
